http/server.go

```diff
@@ -0,0 +1,104 @@
+package http
+
+import (
+	"context"
+	"log"
+	"net"
+	"net/http"
+	"time"
+)
+
+// Server implements a http server which handles request using the configured
+// `http.Handler`.
```

Comment:
Can you document why this is better than plain `*http.Server` (i.e. shutdown etc.)

Reply:
Done